collections/St_Johns_College/SW_V1_St_Johns_College_MS_006.xml

```diff
@@ -0,0 +1,233 @@
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="rubrication">Headings, except the first and last leaves (red),
+                        in text ink.</decoNote>
+                     <decoNote type="decInit">At the head, a 12-line gold leaf capital on blue
+                        flourishing, with a blazon within the letter (see ‘Provenance’), and a full
+                        vinet, a gold leaf and blue bar border.</decoNote>
+                     <decoNote type="flourInit">Similar, but smaller capitals (that for book 3,
+                           <locus from="58vb" to="58vb">fol. 58<hi rend="superscript"
+                           >vb</hi></locus>, only 3 lines) with demivinets at the heads of the
+                        books.</decoNote>
+                     <decoNote type="flourInit">Smaller textual divisions usually blue lombards on
+                        red flourishing.</decoNote>
+                     <decoNote type="paratext">Red running titles with book numbers. See <bibl>AT
+                           no. 595 (58).</bibl></decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <p>A modern replacement. Sewn on six thongs. At the front and rear, one modern
+                        paper flyleaf. A paper tab inside the front cover notes College ownership
+                           <date calendar="Gregorian" notBefore="1800" notAfter="1900">(s.
+                           xix)</date>.</p>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notBefore="1425" notAfter="1475">s. xv
+                        med.</origDate>
+                     <origPlace>
+                        <country key="place_7002445">England</country>
+                     </origPlace>
+                  </origin>
+                  <provenance>Within the initial O <locus from="1ra" to="1ra">(fol. 1<hi
+                           rend="superscript">ra</hi>)</locus>, a blazon, argent (?), a barry of
+                     three, or, on the top argent bar ‘I’affie bien’ and on the second ‘leynthale’
+                     plus a mark. Similarly, in the lower margin, <locus from="50ra" to="50ra">fol.
+                           50<hi rend="superscript">ra</hi></locus>, ‘I’affie bien LEYNTHALE’ plus
+                     mark <date calendar="Gregorian" notBefore="1475" notAfter="1500">(s. xv
+                        ex.)</date>. Apparently to be associated with the arms of <persName
+                        key="person_4618" role="fmo">Sir Rowland Leynthall</persName> of <placeName
+                        key="">Leynthall</placeName> and <placeName key="">Hampton Court
+                        (Herefordshire)</placeName> , who fought at <placeName key=""
+                        >Agincourt</placeName>, but these bear no resemblance to the modern
```

Comment:
I don't think Agincourt needs a key - it is not directly relevant to the provenance of the manuscript and doesn't need to be indexed